lib/host/gen_helpers.js

```diff
@@ -216,10 +215,7 @@ const genHelpers = (state = {}, adapter) => {
         [order.cid + '']: order
       }
 
-      // Note that we don't wait for the order to submit here, since it might
-      // fill immediately triggering a fill event before we patch the state
-      // orders/allOrders objects
-      adapter.submitOrderWithDelay(connection, delay, order).then((order) => { // eslint-disable-line
+      await adapter.submitOrder(connection, order).then((order) => { // eslint-disable-line
```

Comment:
does this work with ping pong orders, submit delay 5seconds?

afaik this would be only possible with step 2 of the task, where we have concurrent order submits?

Reply:
example: order is submitted, confirmation from the backend takes 4 seconds to arrive...

Reply:
yes, confirmed... `submit_all_orders.js` calls `submitOrder` in a blocking loop

Reply:
"await" has been removed to allow Ping/Pong order can create a large number of orders in parallel. 

 In this case, the problem [TWAP: Saving orders on state in each interval leads to numerous orders being submitted at once when internet connnection is restored.](https://app.asana.com/0/1125859137800433/1200162041515069/1200163628599973/f) remains and will be fixed in https://github.com/bitfinexcom/bfx-hf-algo/pull/112, but requires additional changes so as not to break the changes of current PR